scope.go

```diff
@@ -103,6 +129,21 @@ func (s *standardScope) Report(r StatsReporter) {
 	// we do nothing for timers here because timers report directly to ths StatsReporter without buffering
 }
 
+// reportLoop is used by the root scope for periodic reporting
+func (s *standardScope) reportLoop(interval time.Duration) {
+	ticker := time.NewTicker(interval)
```

Comment:
Might be a good idea to close a Scope so we don't leak go-routines once we don't need the scope any more (ie in testing etc)